src/content/guides/development-vagrant.mdx

```diff
@@ -56,13 +56,13 @@ Note that you also need to create an `app.js` file.
 Now, let's run the server:
 
 ```bash
-webpack serve --host 0.0.0.0 --public 10.10.10.61:8080 --watch-poll
+webpack serve --host 0.0.0.0 --client-web-socket-url 10.10.10.61:8080 --watch-poll
```

Comment:
Also, I think we need to update `--watch-poll` to `--watch-options-poll` everywhere.

```suggestion
webpack serve --host 0.0.0.0 --client-web-socket-url 10.10.10.61:8080 --watch-options-poll
```